packages/wonder-blocks-labeled-field/src/components/labeled-field.tsx

```diff
@@ -184,27 +225,40 @@ export default function LabeledField(props: Props) {
 
 const styles = StyleSheet.create({
     label: {
-        color: color.offBlack,
+        color: semanticColor.text.primary,
     },
     lightLabel: {
-        color: color.white,
+        color: semanticColor.text.inverse,
     },
     description: {
-        color: color.offBlack64,
+        color: semanticColor.text.secondary,
     },
     lightDescription: {
         color: color.white64,
     },
+    errorSection: {
+        flexDirection: "row",
+        gap: spacing.xSmall_8,
+    },
     error: {
-        color: color.red,
+        color: semanticColor.status.critical.foreground,
     },
     lightError: {
         color: color.fadedRed,
     },
+    errorIcon: {
+        marginTop: "1px", // This vertically aligns the icon with the text
+    },
+    errorMessage: {
+        minWidth: "0",
+    },
     required: {
-        color: color.red,
+        color: semanticColor.status.critical.foreground,
     },
     lightRequired: {
         color: color.fadedRed,
     },
+    textWordBreak: {
+        overflowWrap: "break-word",
```

Comment:
I added the `overflowWrap` style to all WB Label* components we use in LabeledField so that overflowing text can be wrapped!

I was wondering, should the WB typography components all handle overflowing text? I was thinking we could add it to all typography components, but that would impact a lot of places. Or maybe not everything should wrap the same way? Or some things might need truncation or an ellipsis. Let me know if anyone has any thoughts on this!

Reply:
Is there a concept of utility classes we could use to make this kind of style more portable? That could add flexibility while keeping things consistent (and requiring less CSS repetition).

Reply:
We could create a shared stylesheet like what we have for [typography styles](https://github.com/Khan/wonder-blocks/blob/b8f0fcf91e04bfe8be931b739cea49c6d846d2aa/packages/wonder-blocks-typography/src/util/styles.ts#L20).  Not sure which wonder-blocks package it would go in, maybe `core`? Or a new package called something like `wonder-blocks-styles`? 

I would also like to use something like utility classes or mixins, especially around common state styles (disabled state, hover, focus, etc) and other styles like the `sr-only` styling (related issue: WB-1753) 

I think @jandrade was also considering the concept of utility classes in the CSS Framework investigation!

Reply:
Ooh, yes! That sounds awesome. Either `core` or a new package would make sense to me! I'll be curious to hear @jandrade's thoughts on utility classes, too.

Reply:
I like the idea of `wonder-blocks-styles` a lot :). Specially that a lot of places in `webapp` could benefit from this as well.

`core` could work too, but I feel that, as most of the packages depend on it, it would mean that for every change in the style utils, then all the packages would need to be bumped, and sometimes doesn't make sense due to the current API surface of that package.... as core currently does all sorts of things (e.g. HOCs, hooks, components, styling utils). I've thinking for a while to split `core` into smaller parts, and maybe this could be a good opportunity for moving some of that complexity to the right packages (like `addStyle` could be moved to `wb-styles`, for example).

As for utility classes as a concept, I think we probably don't need to go at the atomic level for now (like `tailwind`), but composed utility classes could work for the current state of Wonder Blocks. Generic things like this example, + other things like form field styles (@beaesguerra suggested this idea a while ago!), etc, could be useful for avoiding repetition.

As for atomic CSS, I've investigated this methodology as a possible approach to take for our CSS styling in the future. You can check `PandaCSS` in this document I wrote some time ago: https://khanacademy.atlassian.net/wiki/spaces/FRONTEND/pages/2966650967/Experimenting+with+CSS+Modules+and+Panda+CSS

Happy to share more details and discuss about it at any moment.

Reply:
Awesome. `wb-styles` and decoupling from core makes a lot of sense! @beaesguerra what do you to implement for this PR to be considered done? Definitely don't want to bikeshed your work on error icons for broader wish list items.

Reply:
Created WB-1776 for the wonder-blocks-style package! I won't include as part of the LabeledField work. Feel free to add any more details or ideas for things we'd like to include!

For this PR, I am still looking into the aria-describedby issue for the error message (from the previous [comment](https://github.com/Khan/wonder-blocks/pull/2344#discussion_r1796197516)). It is proving to be less straightforward than I thought because [VoiceOver doesn't announce aria-describedby info on elements with role="alert"](https://discussions.apple.com/thread/255163557?sortBy=rank) 🙃 so I am investigating alternatives

I'm thinking I'll leave this PR as is to focus on styling and then make updates in the following validation PR to address a11y issues with the errors